File: C/PRAK304-2210817310017-Rohid_Akmal.c
#include <stdio.h>

void main(){
    int a;

    scanf("%d", &a);

    if(a > 0 && a < 10){printf("Satuan");}
    else if(a >= 10 && a < 20){printf("Belasan");}
    else if(a >= 20 && a < 100){printf("Puluhan");}
    else if(a > 99){printf("Anda Menginput Melebihi Limit Bilangan");}
    else if(a == 0){printf("nol");}
}

File: Python/PRAK304-2210817310017-Rohid Akmal.py
a = int(input(""))

if a > 0 and a < 10: print("Satuan")
elif a >= 10 and a < 20: print("Belasan")
elif a >= 20 and a < 100: print("Puluhan")
elif a > 99: print("Anda Menginput Melebihi Limit Bilangan")
elif a == 0: print("nol")